libvmi/accessors.c

```diff
@@ -335,10 +335,10 @@ GSList* vmi_get_va_pages(vmi_instance_t vmi, addr_t dtb) {
 
 addr_t vmi_pagetable_lookup (vmi_instance_t vmi, addr_t dtb, addr_t vaddr)
```

Comment:
That would make it the same as the `_cache()` version, so that could go away.  It's fine with me as long we can get the +1's for changing external API.